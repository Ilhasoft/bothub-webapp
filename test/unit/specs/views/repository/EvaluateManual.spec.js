import { shallowMount, createLocalVue } from '@vue/test-utils';
import RepositoryEvaluate from '@/views/repository/EvaluateManual';
import BH from 'bh';
import Vuex from 'vuex';


const localVue = createLocalVue();
localVue.use(BH);
localVue.use(Vuex);

describe('TestView.vue', () => {
  let wrapper;
  let state;
  let store;
  let actions;
  let getters;
  beforeEach(() => {
    state = {
      selectedRepository: {
        uuid: 'dbe1539c-88ba-43e0-bed5-54729218cec9',
      },
    };
    actions = {
      getEvaluateExample: jest.fn(),
    };
    getters = {
      authenticated: jest.fn(),
      getUpdateRepositoryState: jest.fn(),
    };
    store = new Vuex.Store({
      modules: {
        Repository: {
          state,
          getters,
          actions,
        },
      },
    });
    wrapper = shallowMount(RepositoryEvaluate, { localVue, store });
  });

  test('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('run new test', () => {
    expect(wrapper.vm.newEvaluate()).toBeTruthy();
  });
});
