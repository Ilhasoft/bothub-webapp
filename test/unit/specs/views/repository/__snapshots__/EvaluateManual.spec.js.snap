// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestView.vue renders correctly 1`] = `
<repository-view-base-stub quicktest="true">
  <!---->
  <!---->
</repository-view-base-stub>
`;
